test/fixtures/output/elixir/httpoison/application-form-encoded.ex

```diff
@@ -0,0 +1 @@
+HTTPoison.request(:post, "http://mockbin.com/har", "foo=bar&hello=world", [{"content-type", "application/x-www-form-urlencoded"}])
```

Comment:
Not that it matters a whole lot but is it preferable with this library to do `.post`, `.get`, `.put` or `.request` with a `:method` argument?

Reply:
Even the author of the library uses mixed examples, so it is mostly up to the user's preference and there is no hard restriction. I did it this way since the code generation is a bit simpler, but functionally everything is equivalent.